ArcGISRuntimeSDKQt_CppSamples/Layers/BrowseOGCAPIFeatureService/BrowseOGCAPIFeatureService.h

```diff
@@ -0,0 +1,88 @@
+// [WriteFile Name=BrowseOGCAPIFeatureService, Category=Layers]
+// [Legal]
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifndef BROWSEOGCAPIFEATURESERVICE_H
+#define BROWSEOGCAPIFEATURESERVICE_H
+
+namespace Esri
+{
+namespace ArcGISRuntime
+{
+class Map;
+class MapQuickView;
+}
+}
+
+#include <QObject>
+
+#include "OgcFeatureService.h"
```

Comment:
I think all of these types are stored as pointers so they can be forward declared